docs/reference/modules/terraform-aws-asg/asg-instance-refresh/asg-instance-refresh.md

```diff
@@ -73,6 +76,138 @@ The Terraform [instance_refresh](https://registry.terraform.io/providers/hashico
 *   Auto Scaling Groups support up to one active instance refresh at a time. When this resource is updated, any existing refresh is cancelled.
 *   Depending on health check settings and group size, an instance refresh may take a long time or fail. The Terraform `instance_refresh` resource does not wait for the instance refresh to complete.
 
+## Sample Usage
+
+<ModuleUsage>
+
+```hcl title="main.tf"
+
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY GRUNTWORK'S ASG-INSTANCE-REFRESH MODULE
+# ---------------------------------------------------------------------------------------------------------------------
+
+module "asg-instance-refresh" {
+
+  source = "git::git@github.com:gruntwork-io/terraform-aws-asg.git//modules/asg-instance-refresh?ref=v0.21.1"
+
+  # ---------------------------------------------------------------------------------------------------------------------
+  # REQUIRED VARIABLES
+  # ---------------------------------------------------------------------------------------------------------------------
+
+  # The name of the ASG.
+  asg_name = <INPUT REQUIRED>
+
+  # The desired number of EC2 Instances to run in the ASG initially. Note that auto
+  # scaling policies may change this value. If you're using auto scaling policies to
+  # dynamically resize the cluster, you should actually leave this value as null.
+  desired_capacity = <INPUT REQUIRED>
```

Comment:
NIT: if an input variable has `sensitive = true`, we should probably render it as commented out, and the comment should say "this is a sensitive variable and should prob be set through env vars."

Reply:
Captured this in: https://gruntwork.atlassian.net/browse/CORE-578